SharpPcap/LibPcap/LibPcapSafeNativeMethods.Interop.cs

```diff
@@ -126,7 +126,7 @@ internal static partial class LibPcapSafeNativeMethods
         /// </summary>
         /// <returns></returns>
         [DllImport(PCAP_DLL, CharSet = CharSet.Ansi, CallingConvention = CallingConvention.Cdecl)]
-        internal extern static bool pcap_offline_filter(IntPtr /*bpf_program **/ prog, IntPtr /* pcap_pkthdr* */ header, IntPtr pkt_data);
+        internal extern static int pcap_offline_filter(IntPtr /*bpf_program **/ prog, IntPtr /* pcap_pkthdr* */ header, IntPtr pkt_data);
```

Comment:
This looks like the kind of bug that would cause crashes similar to the WinDivert one, could we have it back-ported to v5.x  ?

Reply:
We could. My time to support the library has been high recently but we've never supported old major revs like we might if there were 10x the users or 2x the developers. And what happens if we release a 5.x release after releasing a 6.x release? So I'm not opposed if you want to figure out all of that stuff. It's just the process of making a 5.x branch at the point where we started on the 6.x changes, if we plan on actually supporting 5.x.

Reply:
the v6.x release is not going to be soon, patching this for v5.x would be nice while we finalize v6.x